components/automate-ui/src/app/pages/automate-settings/automate-settings.component.ts

```diff
@@ -216,8 +219,11 @@ export class AutomateSettingsComponent implements OnInit, OnDestroy {
             this.saving = false;
           } else if (changeConfigurationSelector.status === 'loadingSuccess') {
             this.showSuccessNotification();
-            this.automateSettingsForm.markAsPristine();
             this.saving = false;
+            // After a successful Save, we flip shouldResetValues to true
+            // to update the resetOrigin values
```

Comment:
Consider rewording for clarity (harping on my theme--this is more behavior, less implementation 😁 ):
```suggestion
            // After a successful save, trigger a notification to FormControlDirective
            // to consider the newly saved values as the new "original" values.
```